2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -472,9 +472,9 @@ void Drivetrain::VelocityCLDrive(const frc::DifferentialDriveWheelSpeeds &target
 //
 bool Drivetrain::LoadTrajectory()
 {
-    wpi::SmallString<64> outputDirectory;

```

Comment:
This Drivetrain::LoadTrajectory function no longer does anything; It would be better to just remove it (both remove the definition as well as the spot where we call it)